### PR TITLE
Added more missing functions

### PR DESCRIPTION
**Pull request by mcrispino**
Issue: [78201](https://issues.genexus.com/viewissue.aspx?78201)

Added functions:
- browserID
- browserVersion
- space
- time
- yMDHMStoT

### CODE REVIEW

File: src/datetime/newInstance.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * @param {number} year
+ * @param {number} month
+ * @param {number} day
+ * @param {number} hour
+ * @param {number} minutes
+ * @param {number} seconds
+ * @return Date
+ */
+
+import { DateTime } from "luxon";
```
Comment (ncardeli) [inline src/datetime/newInstance.ts]: Is `DateTime` being used?

File: src/datetime/newInstance.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+export const newInstance = (
+  year: number,
+  month: number,
+  day: number,
+  hour: number,
+  minutes: number,
+  seconds: number
+): Date => {
+  const ret = new Date(year, month - 1, day, hour, minutes, seconds, 0);
+  return ret.getFullYear() === year &&
```
Comment (ncardeli) [inline src/datetime/newInstance.ts]: 👀 interesting way of ensuring the provided date parts resulted in a valid date.

File: src/datetime/newInstance.ts (hunk 1 above)
Comment (ncardeli) [inline src/datetime/newInstance.ts]: Is DateTime being used?

File: src/web/browserCommon.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
+/**
+ * Returns the browser Id and version from the user agent string
+ * @param userAgent
+ */
+export function browserIdFromAgent(userAgent: string): BrowserResult {
+  if (!userAgent) {
+    return { id: BrowserIdValues.UnknownAgent, version: "" };
+  }
+
+  let matches = userAgent.match(/msie\/([0-9\.]+)/i);
```
Comment (ncardeli) [inline src/web/browserCommon.ts]: The regular expressions inside this function create a new Regex instance for each one, every time the function is called.

As an optimizaction, shouldn't we define de regular expressions as constants outside the function?

File: src/datetime/newInstance.ts (hunk 2 above)
Comment (mcrispino) [reply src/datetime/newInstance.ts]: Copied from date/newInstance.ts, credits to @dmendez :)

File: src/datetime/newInstance.ts (hunk 1 above)
Comment (mcrispino) [reply src/datetime/newInstance.ts]: Actually, it isn't. Fixed at [ae9b1b1](https://github.com/genexuslabs/web-standard-functions/pull/77/commits/ae9b1b1960b85aea7790bbcdeb8ee987be139f28)

File: src/web/browserCommon.ts (hunk 3 above)
Comment (mcrispino) [reply src/web/browserCommon.ts]: Fixed at [bae20d5](https://github.com/genexuslabs/web-standard-functions/pull/77/commits/bae20d534cfbf14e2628c474430922d670bd0245)

**Pull request merged**